src/sass/app/components/market.scss

```diff
@@ -288,6 +288,23 @@
                         font-size: 20px;
                         padding: 16px 0px;
                         font-weight: 700;
+                        flex: 1 0 auto;
+                        color: $COLOR_BLUE;
+                        transition: none;
```

Comment:
Can remove this as default value for transition is none?

Reply:
🫡 can